website/docs/best-practices/how-we-style/1-how-we-style-our-dbt-models.md

```diff
@@ -8,6 +8,10 @@ id: 1-how-we-style-our-dbt-models
 - 👥 Models should be pluralized, for example, `customers`, `orders`, `products`.
 - 🔑 Each model should have a primary key.
 - 🔑 The primary key of a model should be named `<object>_id`, for example, `account_id`. This makes it easier to know what `id` is being referenced in downstream joined models.
+- Use underscores for naming dbt models; avoid dots.
+  - ✅  `models_without_dots`
+  - ❌ `models.with.dots`
+  - Underscores are more compatible and clear across platforms. Dots can cause quoting and case sensitivity complexities, especially in systems like Snowflake. Databases may also misinterpret dots as schema or database name separators. [dbt uses dots](https://discourse.getdbt.com/t/why-cant-model-names-contain-dots/422) in its `unique_id` generation to differentiate resource types, package names, and resource names, which is another reason to avoid dots in model name. Read the [discussion](https://github.com/dbt-labs/dbt-core/issues/3246) for more details.
```

Comment:
```suggestion
  - Use underscores because they’re more compatible and clear across platforms. Dots can cause
 dbt Cloud parser, quoting, and case sensitivity issues, especially in systems like Snowflake. Databases may also misinterpret dots as schema or database name separators. [dbt uses dots](https://discourse.getdbt.com/t/why-cant-model-names-contain-dots/422) in its `unique_id` generation to differentiate resource types, package names, and resource names, which is another reason to avoid dots in model name. Read the [discussion](https://github.com/dbt-labs/dbt-core/issues/3246) for more details.
```